b2share/modules/records/serializers/schemas/json.py

```diff
@@ -43,11 +43,15 @@ def filter_internal(self, data):
         if '_files' in data['metadata']:
             del data['metadata']['_files']
         if '_pid' in data['metadata']:
+            epic_pids = [p for p in data['metadata']['_pid']
+                         if p.get('type') == 'handle_pid']
+            if len(epic_pids) > 0:
+                data['metadata']['PID'] = epic_pids[0].get('value')
```

Comment:
It might be confusing for users to have things called just `PID`. Could we name it `handle_pid`? Or is it for backward compatibility?